specifyweb/specify/migrations/0005_stratigraphy_age.py

```diff
@@ -0,0 +1,389 @@
+# Generated by Django 3.2.15 on 2024-08-02 06:04
+
+from django.apps import apps as specify_apps
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+from specifyweb.specify.models import protect_with_blockers
+from specifyweb.specify.update_schema_config import revert_table_field_schema_config, revert_table_schema_config, update_table_field_schema_config_with_defaults, update_table_schema_config_with_defaults
+
+SCHEMA_CONFIG_TABLES = [
+    ('AbsoluteAge', None),
+    ('RelativeAge', None),
+    ('TectonicUnitTreeDef', None),
+    ('TectonicUnitTreeDefItem', None),
+    ('TectonicUnit', None),
+    ('RelativeAgeCitation', None),
+    ('RelativeAgeAttachment', None),
+    ('AbsoluteAgeCitation', None),
+    ('AbsoluteAgeAttachment', None),
+]
+SCHEMA_CONFIG_MOD_TABLE_FIELDS = { # TODO: make schema config corrections in new migration
+    'Collectionobject': ['relativeAges', 'absoluteAges', 'collectionObjectType'],
+    'Collection': ['collectionObjectType'],
+    'Geographytreedef': ['discipline'],
+    'Geologictimeperiodtreedef': ['discipline'],
+    'Lithostrattreedef': ['discipline'],
+}
+PICKLIST_NAME = 'AgeType'
+DEFAULT_AGE_TYPES = [
+    'Sedimentation', 
+    'Metamorphism', 
+    'Erosion', 
+    'Diagenetic', 
+]
+
+def create_agetype_picklist(apps):
+    Collection = apps.get_model('specify', 'Collection')
+    Picklist = apps.get_model('specify', 'Picklist')
+    PicklistItem = apps.get_model('specify', 'Picklistitem')
+    # Create a AgeType picklist for each collection
+    for collection in Collection.objects.all():
+        age_type_picklist, _ = Picklist.objects.get_or_create(
+            name=PICKLIST_NAME,
+            issystem=False,
+            readonly=False,
+            sizelimit=-1,
+            sorttype=1,
+            type=0,
+            collection=collection,
+            formatter=PICKLIST_NAME
+        )
+        for age_type in DEFAULT_AGE_TYPES:
+            PicklistItem.objects.create(
+                title=age_type,
+                value=age_type,
+                picklist=age_type_picklist
+            )
+
+def revert_agetype_picklist(apps, schema_editor):
+    Collection = apps.get_model('specify', 'Collection')
+    Picklist = apps.get_model('specify', 'Picklist')
+    PicklistItem = apps.get_model('specify', 'Picklistitem')
+
+    for collection in Collection.objects.all():
+        age_type_pick_lists = Picklist.objects.filter(name=PICKLIST_NAME, collection=collection)
+
+        for age_type_pick_list in age_type_pick_lists:
+            PicklistItem.objects.filter(picklist=age_type_pick_list).delete()
+            age_type_pick_list.delete()
+
+def create_table_schema_config_with_defaults(apps):
+    Discipline = specify_apps.get_model('specify', 'Discipline')
```

Comment:
Can use `apps` here instead of importing `specify_apps`
```suggestion
def create_table_schema_config_with_defaults(apps):
    Discipline = apps.get_model('specify', 'Discipline')
```